### ISSUE TITLE
CMake CI failing on Windows

### ISSUE THREAD

**Issue by RossBencina** [PortAudio/portaudio#697]
" could not find any instance of Visual Studio."

Presumably GitHub has changed their MSVC version. Here's a log:

```
[Skip to content](https://github.com/PortAudio/portaudio/runs/5225744338?check_suite_focus=true#start-of-content)[](https://github.com/)
Search or jump to…
[Pull requests](https://github.com/pulls)
[Issues](https://github.com/issues)
[Marketplace](https://github.com/marketplace)
[Explore](https://github.com/explore)[](https://github.com/notifications)
 
@RossBencina 
https://github.com/PortAudio
/
[portaudio](https://github.com/PortAudio/portaudio)
Public
[Code](https://github.com/PortAudio/portaudio)
Issues
239
Pull requests
8
[Discussions](https://github.com/PortAudio/portaudio/discussions)
[Actions](https://github.com/PortAudio/portaudio/actions)
[Projects](https://github.com/PortAudio/portaudio/projects?type=beta)
[Wiki](https://github.com/PortAudio/portaudio/wiki)
[Security](https://github.com/PortAudio/portaudio/security)
[Insights](https://github.com/PortAudio/portaudio/pulse)
[Settings](https://github.com/PortAudio/portaudio/settings)
Mac: change some internal errors to host errors. CMake build #484
[Summary](https://github.com/PortAudio/portaudio/actions/runs/1856191512)
Jobs
[Ubuntu GCC](https://github.com/PortAudio/portaudio/runs/5225744212?check_suite_focus=true)
[Ubuntu MinGW](https://github.com/PortAudio/portaudio/runs/5225744267?check_suite_focus=true)
[Windows MSVC](https://github.com/PortAudio/portaudio/runs/5225744338?check_suite_focus=true)
[Windows MinGW](https://github.com/PortAudio/portaudio/runs/5225744391?check_suite_focus=true)
[macOS Clang](https://github.com/PortAudio/portaudio/runs/5225744437?check_suite_focus=true)
[macOS Clang framework](https://github.com/PortAudio/portaudio/runs/5225744484?check_suite_focus=true)
[Windows MSVC](https://github.com/PortAudio/portaudio/runs/5225744338?check_suite_focus=true#logs)
failed 7 days ago in 1m 49s
Search logs
2s
9s
0s
2s
12s
1m 21s
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\runneradmin\AppData\Local\vcpkg\archives\31\310dcac88d37ec7cb56f7b5609f37336033fec6534d3c0e29755e2b68d7c7322.zip
Building package vcpkg-cmake-config[core]:x64-windows... done
Installing package vcpkg-cmake-config[core]:x64-windows...
Installing package vcpkg-cmake-config[core]:x64-windows... done
Elapsed time for package vcpkg-cmake-config:x64-windows: 70.97 ms
Starting package 3/5: jack2:x64-windows
Building package jack2[core]:x64-windows...
-- Downloading https://github.com/jackaudio/jack2/archive/v1.9.19.tar.gz -> jackaudio-jack2-v1.9.19.tar.gz...
-- Extracting source D:/a/portaudio/portaudio/vcpkg/downloads/jackaudio-jack2-v1.9.19.tar.gz
-- Using source at D:/a/portaudio/portaudio/vcpkg/buildtrees/jack2/src/v1.9.19-6f9ce88172.clean
-- Found external ninja('1.10.2').
-- Configuring x64-windows
-- Building x64-windows-dbg
-- Building x64-windows-rel
-- Using msys root at D:/a/portaudio/portaudio/vcpkg/downloads/tools/msys2/aa5af7b2aa7e90e8
-- Fixing pkgconfig file: D:/a/portaudio/portaudio/vcpkg/packages/jack2_x64-windows/lib/pkgconfig/jack.pc
-- Fixing pkgconfig file: D:/a/portaudio/portaudio/vcpkg/packages/jack2_x64-windows/debug/lib/pkgconfig/jack.pc
-- Installing: D:/a/portaudio/portaudio/vcpkg/packages/jack2_x64-windows/share/jack2/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\runneradmin\AppData\Local\vcpkg\archives\23\23612eedd4716e0835b184276e6fa00aadfc81702343[51](https://github.com/PortAudio/portaudio/runs/5225744338?check_suite_focus=true#step:6:51)6a153b441ebda013da.zip
Building package jack2[core]:x64-windows... done
Installing package jack2[core]:x64-windows...
Installing package jack2[core]:x64-windows... done
Elapsed time for package jack2:x64-windows: 19.83 s
Starting package 4/5: pthreads:x64-windows
Building package pthreads[core]:x64-windows...
-- Downloading https://sourceforge.net/projects/pthreads4w//files/pthreads4w-code-v3.0.0.zip/download;https://sourceforge.net/projects/pthreads4w//files/pthreads4w-code-v3.0.0.zip/download?use_mirror=cfhcable;https://sourceforge.net/projects/pthreads4w//files/pthreads4w-code-v3.0.0.zip/download?use_mirror=pilotfiber;https://sourceforge.net/projects/pthreads4w//files/pthreads4w-code-v3.0.0.zip/download?use_mirror=gigenet;https://sourceforge.net/projects/pthreads4w//files/pthreads4w-code-v3.0.0.zip/download?use_mirror=versaweb;https://sourceforge.net/projects/pthreads4w//files/pthreads4w-code-v3.0.0.zip/download?use_mirror=ayera;https://sourceforge.net/projects/pthreads4w//files/pthreads4w-code-v3.0.0.zip/download?use_mirror=netactuate;https://sourceforge.net/projects/pthreads4w//files/pthreads4w-code-v3.0.0.zip/download?use_mirror=phoenixnap;https://sourceforge.net/projects/pthreads4w//files/pthreads4w-code-v3.0.0.zip/download?use_mirror=astuteinternet;https://sourceforge.net/projects/pthreads4w//files/pthreads4w-code-v3.0.0.zip/download?use_mirror=freefr;https://sourceforge.net/projects/pthreads4w//files/pthreads4w-code-v3.0.0.zip/download?use_mirror=netcologne;https://sourceforge.net/projects/pthreads4w//files/pthreads4w-code-v3.0.0.zip/download?use_mirror=deac-riga;https://sourceforge.net/projects/pthreads4w//files/pthreads4w-code-v3.0.0.zip/download?use_mirror=excellmedia;https://sourceforge.net/projects/pthreads4w//files/pthreads4w-code-v3.0.0.zip/download?use_mirror=iweb;https://sourceforge.net/projects/pthreads4w//files/pthreads4w-code-v3.0.0.zip/download?use_mirror=jaist;https://sourceforge.net/projects/pthreads4w//files/pthreads4w-code-v3.0.0.zip/download?use_mirror=jztkft;https://sourceforge.net/projects/pthreads4w//files/pthreads4w-code-v3.0.0.zip/download?use_mirror=managedway;https://sourceforge.net/projects/pthreads4w//files/pthreads4w-code-v3.0.0.zip/download?use_mirror=nchc;https://sourceforge.net/projects/pthreads4w//files/pthreads4w-code-v3.0.0.zip/download?use_mirror=netix;https://sourceforge.net/projects/pthreads4w//files/pthreads4w-code-v3.0.0.zip/download?use_mirror=ufpr;https://sourceforge.net/projects/pthreads4w//files/pthreads4w-code-v3.0.0.zip/download?use_mirror=tenet -> pthreads4w-code-v3.0.0.zip...
-- Extracting source D:/a/portaudio/portaudio/vcpkg/downloads/pthreads4w-code-v3.0.0.zip
-- Applying patch fix-arm-macro.patch
-- Applying patch use-md.patch
-- Using source at D:/a/portaudio/portaudio/vcpkg/buildtrees/pthreads/src/49e541b66c-53502e357c.clean
-- Building x64-windows-rel
-- Building x64-windows-rel done
-- Building x64-windows-dbg
-- Building x64-windows-dbg done
-- Installing: D:/a/portaudio/portaudio/vcpkg/packages/pthreads_x64-windows/share/pthreads/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\runneradmin\AppData\Local\vcpkg\archives\d7\d7e68a2b69bfa2d2d2cb9b0a8adbf07d9caa6b2011662d705a79b95ad504ca64.zip
Building package pthreads[core]:x64-windows... done
Installing package pthreads[core]:x64-windows...
Installing package pthreads[core]:x64-windows... done
Elapsed time for package pthreads:x64-windows: 26.21 s
Starting package 5/5: tre:x64-windows
Building package tre[core]:x64-windows...
-- Downloading https://github.com/laurikari/tre/archive/6fb7206b935b35814c5078c20046dbe065435363.tar.gz -> laurikari-tre-6fb7206b935b35814c5078c20046dbe065435363.tar.gz...
-- Extracting source D:/a/portaudio/portaudio/vcpkg/downloads/laurikari-tre-6fb7206b935b35814c5078c20046dbe065435363.tar.gz
-- Using source at D:/a/portaudio/portaudio/vcpkg/buildtrees/tre/src/e065435363-ca7927034b.clean
-- Found external ninja('1.10.2').
-- Configuring x64-windows
-- Building x64-windows-dbg
-- Building x64-windows-rel
-- Installing: D:/a/portaudio/portaudio/vcpkg/packages/tre_x64-windows/share/tre/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\runneradmin\AppData\Local\vcpkg\archives\60\60ff60331220dcb06e92ef7bbeccda87818bc998b707dc03ecaa59046b2fc0ca.zip
Building package tre[core]:x64-windows... done
Installing package tre[core]:x64-windows...
Installing package tre[core]:x64-windows... done
Elapsed time for package tre:x64-windows: 4.[52](https://github.com/PortAudio/portaudio/runs/5225744338?check_suite_focus=true#step:6:52) s

Total elapsed time: [56](https://github.com/PortAudio/portaudio/runs/5225744338?check_suite_focus=true#step:6:56).22 s

-- Running vcpkg install - done
CMake Error at CMakeLists.txt:2 (project):
  Generator

    Visual Studio 16 2019

  could not find any instance of Visual Studio.
-- Configuring incomplete, errors occurred!

See also "D:/a/portaudio/portaudio/build/CMakeFiles/CMakeOutput.log".


Error: Process completed with exit code 1.
```

**Comment by Be-ing** [comment]
Fixed in #698

**Comment by RossBencina** [comment]
Closing as fixed.
